winlogbeat/docs/getting-started.asciidoc

```diff
@@ -150,6 +150,26 @@ Before starting {beatname_uc}, modify the user credentials in
 +{beatname_lc}.yml+ and specify a user who is
 <<privileges-to-publish-events,authorized to publish events>>.
 
+By default, the `winlogbeat` service runs as `Local System` account. If you want to run `winlogbeat` service as a local user account which is not an Administrator, follow the below steps. By following these steps, your local user account will be granted `Log on as a service` security policy and made part of group `Event Log Readers` to read the event log. (You can skip these steps if you want to continue running Winlogbeat as an Administrator.)
```

Comment:
```suggestion
By default, the `{beatname_uc}` service runs as the `LocalSystem` account. If you want to run the `{beatname_uc}` service as a local user account that is not an Administrator, then follow the below steps. The local user account must be granted `Log on as a service` in the security policy and be made part of the `Builtin\Event Log Readers` group to read the event log.
```

I changed to account names to match the names in https://learn.microsoft.com/en-us/windows-server/identity/ad-ds/manage/understand-security-identifiers.

Reply:
Updated as suggested